phpswitch.sh

```diff
@@ -43,19 +62,21 @@ then
 			fi
 		done
 		brew link "$php_version"
-		echo "You will need sudo power from now on"
-		echo "Switching your apache conf"
-		for j in ${php_installed_array[@]}
-		do
-			sudo sed -i.bak "s/^LoadModule[ \t]php5_module[ \t]$php_opt_path$j$php_lib_path/\#LoadModule php5_module $php_opt_path$j$php_lib_path/g" $apache_conf_path
-		done
-		sudo sed -i.bak "s/^\#LoadModule[ \t]php5_module[ \t]$php_mod_path/LoadModule php5_module $php_mod_path/g" $apache_conf_path
-		echo "Restarting apache"
-		sudo apachectl restart
+		if [[ $apache_change -eq 1 ]]; then
+			echo "You will need sudo power from now on"
+			echo "Switching your apache conf"
+			for j in ${php_installed_array[@]}
+			do
+				sudo sed -i.bak "s/^LoadModule[ \t]php5_module[ \t]$php_opt_path$j$apache_php_lib_path/\#LoadModule php5_module $php_opt_path$j$apache_php_lib_path/g" $apache_conf_path
+			done
+			sudo sed -i.bak "s/^\#LoadModule[ \t]php5_module[ \t]$apache_php_mod_path/LoadModule php5_module $apache_php_mod_path/g" $apache_conf_path
+			echo "Restarting apache"
+			sudo apachectl restart
+		fi
 		echo "All done!"
 	else
-		echo "Sorry, but $php_version is not installed via brew. Install by running brew install $php_version"
+		echo "Sorry, but $php_version is not installed via brew. Install by running: brew install $php_version"
 	fi
 else
-	echo "Unknown version of PHP. PHP Switcher can only handle arguements of 53,54,55"
+	echo "Unknown version of PHP. PHP Switcher can only handle arguments of:" ${php_array[@]}
```

Comment:
brew-php-switcher doesn't accept the arguments of php53,php54 etc.

Of course it could but probably better to keep the 53,54,55,56 arguments but tidy them up into an array.